src/pages/search/SearchResults.tsx

```diff
@@ -30,29 +34,37 @@ const SearchResults: FC<SearchResultsProps> = ({ publications, searchTerm }) =>
       {searchTerm && t('results', { count: publications.length, term: searchTerm })}
       <List>
         {publications &&
-          publications.slice(validPage * rowsPerPage, validPage * rowsPerPage + rowsPerPage).map((publication) => (
-            <PublicationListItemComponent
-              key={publication.identifier}
-              primaryComponent={
-                <MuiLink component={Link} to={`/publication/${publication.identifier}/public`}>
-                  {publication.mainTitle}
-                </MuiLink>
-              }
-              secondaryComponent={
-                <Typography component="span">
-                  {new Date(publication.modifiedDate).toLocaleDateString()}
-                  <br />
-                  {publication.contributors.map((contributor) => (
-                    <Fragment key={contributor.identifier}>
-                      <MuiLink component={Link} to={`/user/${contributor.identifier}`}>
-                        {contributor.name}
-                      </MuiLink>{' '}
-                    </Fragment>
-                  ))}
-                </Typography>
-              }
-            />
-          ))}
+          publications.slice(validPage * rowsPerPage, validPage * rowsPerPage + rowsPerPage).map((publication) => {
+            const publicationId = publication.id?.split('/').pop();
```

Comment:
hva er `id`-feltet? trodde Publication bare hadde `identifier` som er UUID

Reply:
og er det nullable?

Reply:
før var det identifier, men nå heter det id. og iden er URI. Ikke egentlig nullable, men måtte gjøre det for å unngå krasj pga gamle publikasjoner